frontends/concrete-python/concrete/fhe/extensions/hint.py

```diff
@@ -32,7 +41,15 @@ def hint(x: Union[Tracer, Any], *, bit_width: Optional[int] = None) -> Union[Tra
     if not isinstance(x, Tracer):  # pragma: no cover
         return x
 
+    bit_width_hint = 0
```

Comment:
should we reject invalid value ?
E.g. bit_width = 0 is ignored.

Reply:
It's the semantically correct behavior. `bit_width = 0` in this context means bit width needs to be **at least** 0 which will always be the case so we can ignore the constraint as it's already satisfied.